### DISCUSSION TITLE
Animated image(gif) not saved successfully for certain website

### DISCUSSION THREAD

**Discussion by windtoostrong** [gildas-lormeau/SingleFile#1602] [Q&A]
**Describe the bug**
Animated image(gif) not saved successfully for certain website

**To Reproduce**
Default configuration, try to save https://www.douban.com/group/topic/272043288/

**Expected behavior**
The html is generated successfully but the saved image is the first frame but not the animated gif image. Is this expected?

**Screenshots**

**Environment**
env: both mac and windows
version: 1.22.72

**Additional context**
Not sure if this is due to how douban renders the gif image, I saw it's using a video player for each animated image from the html code.

**Comment by gildas-lormeau** [comment]
This is not a GIF actually. It's a video. To save it, you have to disable the option `Network >  blocked resources > videos` n SingleFile. Then, you have to use the context menu to play the video when viewing the saved page.

**Comment by windtoostrong** [comment]
Solved my problem, it works, thanks!
